cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -107,7 +107,7 @@ func ResyncPeriod(s *options.CMServer) func() time.Duration {
 func Run(s *options.CMServer) error {
 	// To help debugging, immediately log version
 	glog.Infof("Version: %+v", version.Get())
-	if err := s.Validate(KnownControllers(), ControllersDisabledByDefault.List()); err != nil {
```

Comment:
yes, ControllersDisabledByDefault is default-off controllers. The Validate() function not used the input-parameter, I think can delete it.